regression/cbmc/pointer-primitive-check-05/main.c

```diff
@@ -0,0 +1,13 @@
+#include <stdlib.h>
+
+int main()
+{
+  unsigned char s;
```

Comment:
I guess using an unsigned char limits the size here. Should we be using a size_t that is assumed to be less than max_malloc_size ... or maybe we don't even need that since malloc should just return null if we ask for something too big.

Reply:
Yes, this was meant to limit the size and avoid running into those other problems. But, as you said, it should actually be ok to face these problems. So I've revised this back to your original example and use `size_t`.